dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-dm/src/main/java/org/apache/dolphinscheduler/plugin/datasource/dm/param/DmConnectionParam.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.datasource.dm.param;
+
+import org.apache.dolphinscheduler.spi.datasource.BaseConnectionParam;
+
+public class DmConnectionParam extends BaseConnectionParam {
+
+    @Override
+    public String toString() {
+        return "DmConnectionParam{"
+                + "user='" + user + '\''
+                + ", password='" + password + '\''
+                + ", address='" + address + '\''
```

Comment:
IMHO, remove `password` in `toString()` should be better, this prevents the password from being printed to the log file. WDYT.

Reply:
ok, I'll remove password in toString()

Reply:
The `toString()` method of the `OracleConnectionParam`、`MySQLConnectionParam `class has the password attribute printed.I modified it according to the mysql and oracle specifications。Now I need to remove `password `in `toString()`.

Reply:
> The `toString()` method of the `OracleConnectionParam`、`MySQLConnectionParam `class has the password attribute printed.I modified it according to the mysql and oracle specifications。Now I need to remove `password `in `toString()`?

Please remove all `password` in those classes, also, consider use `lombok` `ToString(excludes = ` to simplify the codes

Reply:
ok, I'll remove `password` in `toString()`